jcl/j9modules.xml

```diff
@@ -38,6 +38,12 @@
 			<case value="java.management">
 				<removeexport package="com.ibm.lang.management" />
 				<removeexport package="com.ibm.virtualization.management" />
+				<removeexport package="com.ibm.java.lang.management.internal" />
```

Comment:
Can `exports com.ibm.java.lang.management.internal to jdk.management;` be removed as well?

Reply:
This is to remove the export statement containing `com.ibm.management` during composing. The export statement without the old package name is added back in by `module-info.java.extra`.